packages/web3-utils/package.json

```diff
@@ -1,6 +1,5 @@
 {
 	"name": "web3-utils",
-	"sideEffects": false,
```

Comment:
What was the need to remove this, please? it was put to support tree shaking (https://webpack.js.org/guides/tree-shaking/) write?

Reply:
reverted